### ISSUE TITLE
website not working

### ISSUE THREAD

**Issue by skgupta507** [ansonbenny/Music-Platform#2]
hello @ansonbenny 
i tried to deploy the site on netlify but i got a error "error facing issues". Ijust separated the client and server folder into different repositories. and hosted the client directory o n netlify and the server directory on adaptible.io and addded the url into the client .env file Creating a new value called Backend_url. Even after doing this iam getting the same issue.
Am i doing something wrong 
can u guide me through the process pls.

i looked at ur hosted site on render after signup the songs only play for 30 seconds i thought i should mention it so that u ca fix it

my hosted site url "https://music-streamerx.netlify.app/" 

u have some other projects also in the same server and client directory type just guide me how can i deploy them.

i will be waiting for ur reply.

**Comment by ShaikHaaris** [comment]
Yes Even i'm facing the same issue. Can you look into these please.if possible

**Comment by ansonbenny** [comment]
It doesn't work on netlify

**Comment by skgupta507** [comment]
Hey @ansonbenny should I deployed it to render or vercel. If not can u recommend which service I should use. And the song on ur site are playing only for 30 secs.

**Comment by ShaikHaaris** [comment]
Brother please guide me. i need your project. and i'm a fresher recent passedout i nee that .enj file with all the screat id and all please kindly share ur other social media wedsite. so that i can reach you there. please help me @ansonbenny please

**Comment by ansonbenny** [comment]
@skgupta507 Spotify doesn't provide full song, i suggest to host on render. before hosting on render, build client and move dist folder to server folder then host server folder on render. then will work both client and server.

**Comment by ansonbenny** [comment]
@ShaikHaaris follow readme steps for setup .env, you can generate MAIL_SECRET from google app password, for spotify id & secret https://developer.spotify.com/documentation/web-api/tutorials/getting-started

**Comment by skgupta507** [comment]
> @skgupta507 Spotify doesn't provide full song, i suggest to host on render. before hosting on render, build client and move dist folder to server folder then host server folder on render. then will work both client and server.

hey @ansonbenny sorry for disturbing but there is only public and src folders in the client directory which folder shuld i move to server from this both.

**Comment by ansonbenny** [comment]
@skgupta507 first you should **build client side**, then you can see a **dist** folder inside client folder then move **dist** folder to server

**Comment by ansonbenny** [comment]
@ShaikHaaris You can connect me on LinkedIn https://in.linkedin.com/in/anson-benny

**Comment by skgupta507** [comment]
> @skgupta507 first you should **build client side**, then you can see a **dist** folder inside client folder then move **dist** folder to server

hey @ansonbenny  do i have to build it locally first and then deploy it to render separately 

like server folder sperately and client floder separately cause in the .env file there is no option to add backend url how will we connect to the backend to client deployment

**Comment by ansonbenny** [comment]
@skgupta507 yes, run this command on client side **npm run build**
